Cargo.toml

```diff
@@ -11,7 +11,6 @@ crate-type = ["cdylib"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-proc-macro2 = "1.0.29"
```

Comment:
I've added this to develop during my first attempts to setup CI/CD. It's no longer required.